velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -350,11 +376,67 @@ TEST_F(HdfsFileSystemTest, multipleThreadsWithFileSystem) {
   }
 }
 
+TEST_F(HdfsFileSystemTest, missingWriteFileViaFileSystem) {
+  try {
+    facebook::velox::filesystems::registerHdfsFileSystem();
+    auto memConfig =
+        std::make_shared<const core::MemConfig>(configurationValues);
+    auto hdfsFileSystem =
+        filesystems::getFileSystem(fullDestinationPath, memConfig);
+    auto writeFile = hdfsFileSystem->openFileForWrite(
+        "hdfs://localhost:7777/path/that/does/not/exist");
+    FAIL() << "expected VeloxException";
+  } catch (facebook::velox::VeloxException const& error) {
```

Comment:
Use VELOX_ASSERT_THROW

Reply:
VELOX_ASSERT_THROW is under velox/common/base/GTestUtil.h,  and it can not be directly include. The unit tests in HdfsFileSystemTest.cpp are all use this try... catch... EXPECT_THAT to test error messages. So I think it not suitable to use VELOX_ASSERT_THROW

Reply:
> it can not be directly include. 

Why not?